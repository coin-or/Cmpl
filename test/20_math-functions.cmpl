parameters:
	 k:=35;
 	 c[1] := sqrt( (k+1) ); 
	 c[2] := exp(10);  
	 c[3] := ln(10);  
	 c[4] := lg(10000);  
	 c[5] := ld(8);  
	 c[6] := 10;  
	 c[7] := sin(2.5);
	 c[8] := cos(7.7);
	 c[9] := tan(10.1);
	 c[10] := acos(0.1);
	 c[11] := asin(0.4);
	 c[12] := atan(1.1);
	 c[13] := sinh(10);
	 c[14] := cosh(3);
	 c[15] := tanh(15);
	 c[16] := abs(-12.55);
	 c[17] := ceil(12.55);
	 c[18] := floor(-12.55);
	 c[19] := round(12.4);
	 
	 { j:=1(1)dim(c[]) : echo "index " + j + ": "+ c[j]; }
	 	 

