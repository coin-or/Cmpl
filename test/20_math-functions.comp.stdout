index 1: 6.000000
index 2: 22026.465795
index 3: 2.302585
index 4: 4.000000
index 5: 3.000000
index 6: 10
index 7: 0.598472
index 8: 0.153374
index 9: 0.800789
index 10: 1.470629
index 11: 0.411517
index 12: 0.832981
index 13: 11013.232875
index 14: 10.067662
index 15: 1.000000
index 16: 12.550000
index 17: 13.000000
index 18: -13.000000
index 19: 12.000000
